resources/user_privilege.rb

```diff
@@ -0,0 +1,22 @@
+#
+# Author:: Jared Kauppila (<jared@kauppi.la>)
+# Cookbook:: windows
+# Resource:: user_privilege
+#
+
+property :principal, String, name_property: true, required: true
```

Comment:
Using name_property and required together doesn't really make sense. name_property allows you to make a property and alternative to the resource's own name. If it's required it's not really an alternative and the name_property part would never actually kick in.

Reply:
Sure, I'll get this updated